### PR TITLE
[ATP-294] Implement work manager

### PR DESCRIPTION
**Pull request by sdvkit**
Implement work manager to the project to send notifications.

https://github.com/sdvkit/PexelsApp/assets/101420114/1b6853de-c277-4eb7-8fff-a65502ee293b

https://github.com/sdvkit/PexelsApp/assets/101420114/7e4f1691-af24-41bc-978d-84a004c88098

https://github.com/sdvkit/PexelsApp/assets/101420114/1b57c4fd-9b63-4d1b-b210-30e43f72e8be

https://github.com/sdvkit/PexelsApp/assets/101420114/8179dd6b-da62-4a80-964e-47f5c2c5474a

### CODE REVIEW

File: app/src/main/java/com/sdv/kit/pexelsapp/presentation/worker/DailyCheckPhotosWorker.kt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    appContext: Context,
+    params: WorkerParameters
+) : CoroutineWorker(appContext, params) {
+
+    override suspend fun doWork(): Result {
+        val notificationManager = NotificationManagerImpl(context = applicationContext)
+        val pendingIntent = buildPendingIntent()
+
+        notificationManager.sendNotification(
+            notificationId = 2,
```
Comment (VladislavShi) [inline app/src/main/java/com/sdv/kit/pexelsapp/presentation/worker/DailyCheckPhotosWorker.kt:22]: by the way lets create some constants, where we can store those ids

Comment (sdvkit) [reply app/src/main/java/com/sdv/kit/pexelsapp/presentation/worker/DailyCheckPhotosWorker.kt:22]: Ok

File: app/src/main/java/com/sdv/kit/pexelsapp/data/manager/NotificationManagerImpl.kt [hunk 2]
```diff
@@ -44,9 +45,4 @@ class NotificationManagerImpl(private val context: Context) : NotificationManage
 
         notificationManager.notify(notificationId, notificationBuilder.build())
     }
-
-    companion object {
```
Comment (VladislavShi) [inline app/src/main/java/com/sdv/kit/pexelsapp/data/manager/NotificationManagerImpl.kt]: Return this to companion object, notification constants we need in one place for better understanding which constant for what notification

**Review by VladislavShi — CHANGES_REQUESTED** [review]

**Review by VladislavShi — APPROVED** [review]
All good for me

**Pull request merged**